src/js/App/Feedback/Feedback.js

```diff
@@ -18,6 +18,14 @@ const Feedback = ({ user }) => {
   const isAvailable = env === 'prod' || env === 'stage';
   const setIsModalOpen = (...args) => dispatch(toggleFeedbackModal(...args));
 
+  const addFeedbackTag = () => {
+    if (window.insights.chrome.isProd) {
```

Comment:
Can you create a helper function that will give you the same value? We want to stop using the window API.

You can check this file to get the implementation: https://github.com/RedHatInsights/insights-chrome/blob/master/src/js/chrome/entry.js#L112

Reply:
Hmm I'm a bit confused by this. Looking at the implementation of that file, getApp is also using the window API.

Reply:
Yup, there is a plenty of parts that are using the window. We want to migrate away from it.

Reply:
I've changed the implementation. Let me know if this is what you meant.